Assets/Scripts/Controllers/CharacterController-DESKTOP-7IQNCQ4.cs

```diff
@@ -28,8 +31,15 @@ public void OnMove(InputValue value)
         _movement = value.Get<Vector2>();
     }
 
+    public bool onGround()
```

Comment:
`public bool onGround()` -> `public bool OnGround()`

PascalCase for class methods.